src/services/ReminderEmailProcessor.ts

```diff
@@ -0,0 +1,83 @@
+import { Response, GenericServerError } from "../utils/Response";
```

Comment:
I'm wondering if we were asked to check if a specific user was sent a reminder email would we be able to do that with the logs we have/without looking in the DB? I can't see anywhere where we are logging the sessionId

Reply:
Added 
this.logger.info("Reminder email sent to user: ", { sessionId })